main.js

```diff
@@ -0,0 +1,64 @@
+// Exercise 1:
+// Way 1:
+
+const isEqual = (arr1, arr2) => {
+  return (JSON.stringify(arr1) === JSON.stringify(arr2)) ? true : false
+}
```

Comment:
1. Không được viết các lệnh dạng:
```javascript
  const condition = true
  if (condition) {
    return true
  } else {
    return false
  }
```

return thẳng luôn cái `condition`

2.
Vd:
```javascript
  const arr1 = [1, 2]
  const arr2 = [2, 1]
  
  // sau khi stringify
  const str1 = "[1, 2]"
  const str2 = "[2, 1]"
```
So sánh vẫn phải trả về `arr1` is equal `arr2`

Nếu stringify như bên trên sẽ bị sai